specs/yara/yara_events.table

```diff
@@ -1,5 +1,5 @@
 table_name("yara_events")
-description("Track YARA matches for files specified in configuration data.")
+description("Historical YARA matches for files and PIDs as specified in configuration.")
```

Comment:
Hrm. IIRC this is an evented table basically identical to the way file events works. We describe that one as `Track time/action changes to files specified in configuration data`

I'm a little leery of "historical" since evented tables don't really store history. (Well, they might. But usually they drop it after it's queried)

Reply:
I reverted this change. How about just merging the explanation for the yara table?

Reply:
It's sorta "historical". Evented tables store events, and then usually purge them out when queried. Generally they're not good for thigns like "show me last tuesday", they're for things like "how do I capture all the processing that are running, since some are shorter lived than my query interval"